.github/workflows/publish-release.yml

```diff
@@ -27,3 +27,26 @@ jobs:
       - uses: MetaMask/action-publish-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          yarn setup
+          yarn build
+      - uses: actions/cache@v2
+        id: restore-build
+        with:
+          path: ./*
```

Comment:
I was expecting this to just cache the build. It looks like the entire repo, including all node deps, are cached as well. Is that necessary?

Reply:
maybe just adding a negate for `node_modules` is good enough here?